assets/js/components/Header.js

```diff
@@ -117,6 +119,8 @@ function Header( { children, subHeader, showNavigation } ) {
 
 			{ showNavigation && <DashboardNavigation /> }
 
+			{ isDashboard && ! isViewOnly && <SubtleNotifications /> }
+
```

Comment:
`isDashboard` is included as condition so it doesn't render in other pages, like settings, etc

Reply:
Had to revert the `isDashboard` check back to header, it does not behave as expected when check is added in component, I guess that is the reason we don;t use it outside the main and entity dashboards